index.html

```diff
@@ -1,889 +1,1433 @@
+            >
+          </p>
+          <p class="about">
+            hi, I'm Amanda, I'm starting my studies in programming. It's still a
+            new world for me, but I'm really enjoying it. I work at Rocketseat,
+            the largest programming school in Latin America, and I have contact
+            with programming every day.
+          </p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a
+                  href="https://www.youtube.com/watch?v=DemagUML778&t=72s"
+                  target="_blank"
+                  title="How to organize a study schedule?"
+                  >Organize a study</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://www.youtube.com/watch?v=QfcozcbDhNM"
+                  target="_blank"
+                  title="Study techniques and learning techniques"
+                  >Study techniques</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://www.youtube.com/watch?v=Fzwr1JcqFo4&t=76s"
+                  target="_blank"
+                  title="Programming study techniques"
+                  >Programming techniques</a
+                >
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!--lenalightning's card end--!>
 
-            <!-- <<<<< Do not change anything below this line >>>>> -->
-            <!-- -------------------------------------------------- -->
+                  <!--Gourav's card-->
 
+        <div class="card">
+          <p class="name">Ashu Mishra</p>
+          <p class="contact">
+            <i class="fab fa-linkedin-in"></i>
+            <a href="https://www.linkedin.com/in/ashu81" target="_blank"
+              >Ashu Mishra</a
+            >
+          </p>
+          <p class="about">
+            Hey! I am Ashu Mishra. I'm a Full Stack Developer.I love programming
+            and making random web-based applications for fun.
+          </p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a
+                  href="https://www.w3schools.com/"
+                  target="_blank"
+                  title="First Resource"
+                  >w3Schools</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://www.youtube.com/c/WebDevSimplified"
+                  target="_blank"
+                  title="Second Resource"
+                  >WebDevSimplified</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://www.youtube.com/c/Freecodecamp"
+                  target="_blank"
+                  title="Third resource"
+                  >Freecodecamp</a
+                >
+              </li>
+            </ul>
+          </div>
```

Comment:
ig this is your card so it is duplicate the other cards so i am closing this pr and pls re open pr with different resources + pls do not format html file keep it as it is to track your changes